packages/babel-traverse/src/path/inference/inferers.js

```diff
@@ -89,7 +89,15 @@ export function LogicalExpression() {
   ]);
 }
 
-export function ConditionalExpression() {
+export function ConditionalExpression(node) {
+  console.log(node);
+  if (t.isTSTypeAnnotation(node)) {
```

Comment:
`isTSTypeAnnotation` is not working here

<details>
<summary>The tree example for <a href="https://github.com/babel/babel/pull/11378/files#diff-15a026b575d179b28985d330d461ff64R1">bug case</a></summary>

```javascript
{
        type: 'ConditionalExpression',
        start: 29,
        end: 42,
        loc: 
         SourceLocation {
           start: Position { line: 2, column: 12 },
           end: Position { line: 2, column: 25 } },
        test: 
         Node {
           type: 'NumericLiteral',
           start: 29,
           end: 30,
           loc: SourceLocation { start: [Object], end: [Object] },
           extra: { rawValue: 1, raw: '1' },
           value: 1,
           leadingComments: undefined,
           innerComments: undefined,
           trailingComments: undefined },
        consequent: 
         Node {
           type: 'CallExpression',
           start: 33,
           end: 36,
           loc: SourceLocation { start: [Object], end: [Object] },
           callee: 
            Node {
              type: 'Identifier',
              start: 33,
              end: 34,
              loc: [Object],
              name: 'a',
              leadingComments: undefined,
              innerComments: undefined,
              trailingComments: undefined },
           arguments: [],
           leadingComments: undefined,
           innerComments: undefined,
           trailingComments: undefined,
           typeParameters: null },
        alternate: 
         Node {
           type: 'CallExpression',
           start: 39,
           end: 42,
           loc: SourceLocation { start: [Object], end: [Object] },
           callee: 
            Node {
              type: 'Identifier',
              start: 39,
              end: 40,
              loc: [Object],
              name: 'b',
              leadingComments: undefined,
              innerComments: undefined,
              trailingComments: undefined },
           arguments: [],
           leadingComments: undefined,
           innerComments: undefined,
           trailingComments: undefined,
           typeParameters: null },
        leadingComments: undefined,
        innerComments: undefined,
        trailingComments: undefined }
```
</details>

Reply:
Now I check `consequent` and `alternate` to be `TSTypeAnnotation`, so it's working fine